### ISSUE TITLE
Elevated urine deoxypyridinoline concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6029]
**Preferred term label:** Elevated urine deoxypyridinoline concentration

**Synonyms** Elevated urine DPD concentration

**Definition (free text, please give PubMed ID)** Concentration of deoxypyridinoline in the urine is above the normal range., PMID: 9127470; PMID: 12661898

**Parent term (use hpo.jax.org/app)** Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
